packages/dds/map/package.json

```diff
@@ -83,6 +83,7 @@
 		"@fluid-tools/build-cli": "^0.20.0",
 		"@fluidframework/build-common": "^1.2.0",
 		"@fluidframework/build-tools": "^0.20.0",
+		"@fluidframework/build-tools": "^0.20.0",
```

Comment:
FYI, looks like this change duplicated entries in a number of package.jsons @curtisman